QUARKUS-5857.md

```diff
@@ -0,0 +1,42 @@
+# Quarkus-5857 WebSockets Next: Allow to send authorization headers from web browsers using JavaScript clients
+
+Jira: https://issues.redhat.com/browse/QUARKUS-5857
+
+Upstream PR: https://github.com/quarkusio/quarkus/pull/45809
+
+## Scope of testing
+Verifying the functionality that allows JavaScript WebSocket clients running in web browsers to send authorization headers to a Quarkus WebSocket Next server.
+This includes:
+- Verify that web-browser-based JavaScript WebSocket clients can correctly format and send authorization (Bearer tokens) via the Sec-WebSocket-Protocol header
+during the WebSocket handshake.
+- Ensure that the Quarkus WebSocket Next server can receive, process, and validate the authorization information sent by web-browser clients for authentication 
+and authorization purposes.
+- Verify also that programmatic WebSocket clients (such as Vert.x WebSocketClient) can send authorization information in the expected format and that the server processes it correctly.
+- Testing both positive and negative authorization scenarios using web-browser and programmatic clients, ensuring that missing, invalid, or unauthorized Bearer tokens result in connection rejection.
+
+### Test cases
+- Successful Authorization via Sec-WebSocket-Protocol header (Web browser client - Automated with Playwright)
+- Failed Authorization - Missing Sec-WebSocket-Protocol header (Web browser client - Automated with Playwright)
+- Failed Authorization - Invalid Bearer Token Format
+- Failed Authorization - Unauthorized Bearer Token
+- Successful Authorization via Sec-WebSocket-Protocol Header (Programmatic Client with Vert.x WebSocketClient)
+- Failed Authorization - Missing Sec-WebSocket-Protocol Header (Programmatic Client with Vert.x WebSocketClient)
+- Verify Correct Handling Post-Successful Authorization (Web-browser Client - Automated with Playwright)
+
+## Getting familiar with the feature
+Getting familiar with
+- Websocket next guide: https://quarkus.io/guides/websockets-next-tutorial
+- Websocket next reference: https://quarkus.io/version/main/guides/websockets-next-reference
+
+## Existing test coverage
+QE TS already has tests for websocket-classic and websocket-next introduced by QUARKUS-5870.md.
```

Comment:
`websocket-classic` is innovative idea but I have never seen current websocket extension based on the undertow to be referred to this way.